### PR TITLE
use equals to compare floats in FloatTransform2D

### PR DESCRIPTION
**Pull request by sebastianbaginski**
@marcusnaslund review

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Justice and *equals*ity for all.

**Comment by marcusnaslund** [comment]
Android testing timed out, can you verify this works on a phone?

**Comment by sebastianbaginski** [comment]
Seems to work on nexus 6

File: source/geometry/FloatTransform2D.ooc [hunk 1]
```diff
@@ -29,15 +29,14 @@ FloatTransform2D: cover {
...
 	scalingY ::= (this d * this d + this e * this e) sqrt()
 	rotationZ ::= this b atan2(this a)
-	isProjective ::= this determinant != 0.0f
-	isAffine ::= this c == 0.0f && this f == 0.0f && this i == 1.0f
+	isProjective ::= !this determinant equals(0.0f)
+	isAffine ::= this c equals(0.0f) && this f equals(0.0f) && this i equals(1.0f)
 	isSimilarity ::= this == This create(this translation, this scaling, this rotationZ)
 	isEuclidian ::= this == This create(this translation, this rotationZ)
-	isIdentity ::= (this a == 1.0f && this e == 1.0f && this i == 1.0f) && (this b == 0.0f && this c == 0.0f && this d == 0.0f && this f == 0.0f && this g == 0.0f && this h == 0.0f)
+	isIdentity ::= (this a equals(1.0f) && this e equals(1.0f) && this i equals(1.0f)) && (this b equals(0.0f) && this c equals(0.0f) && this d equals(0.0f) && this f equals(0.0f) && this g equals(0.0f) && this h equals(0.0f))
```
Comment (marcusnaslund) [inline source/geometry/FloatTransform2D.ooc]: Again two pairs of parentheses that serve no purpose. Would you please remove them?

Comment (sebastianbaginski) [inline source/geometry/FloatTransform2D.ooc]: Done.

**Comment by marcusnaslund** [comment]
Thank you.

**Pull request merged**